python/xorbits/_mars/dataframe/base/accessor.py

```diff
@@ -50,17 +50,29 @@ class StringAccessor:
     dtype: object
     """
 
-    def __init__(self, series):
-        self._series = series
+    _entity = None
+
+    def __init__(self, entity):
+        self._entity = entity
 
     @classmethod
     def _gen_func(cls, method):
         @wraps(getattr(pd.Series.str, method))
         def _inner(self, *args, **kwargs):
-            op = SeriesStringMethod(
-                method=method, method_args=args, method_kwargs=kwargs
+            from ..core import INDEX_TYPE, OutputType
+
+            output_type = None
```

Comment:
This line is not needed.

Reply:
all comments are addressed in the upcoming commits